plugin/aws/ec2Instance.go

```diff
@@ -186,6 +132,13 @@ func (inst *ec2Instance) List(ctx context.Context) ([]plugin.Entry, error) {
 		entries = append(entries, latestConsoleOutput)
 	}
 
+	entries = append(entries, &ec2InstanceSSM{
+		EntryBase:        plugin.NewEntry("ssm"),
```

Comment:
This is weird, because SSM is not a "thing" that belongs to an EC2 instance. Could we remove this?

Reply:
I guess I can remove it completely. Wanted to keep it around to experiment with somehow; it should probably end up being a config and I don't want to deal with the config yet.

Reply:
We could always prefix unused files with `_` ?